sdk/cosmos/src/consistency_level.rs

```diff
@@ -1,4 +1,5 @@
 use crate::headers;
+use crate::prelude::CreateUserResponse;
```

Comment:
I was unsure what to do here, I noticed in Ryan's example PR for converting to pipeline he removed the implement_from! for GetDatabaseResponse but in this case it was still required by examples/permission_00 @ line 66.